docs/standard/choosing-core-framework-server.md

```diff
@@ -89,6 +89,8 @@ Some .NET Framework technologies are not available in .NET Core. Some of them wi
 
 * Workflow related services: Windows Workflow Foundation (WF), Workflow Services (WCF + WF in a single service) and WCF Data Services (formerly known as “ADO.NET Data Services”) are only available on the .NET Framework and there are no plans to bring them to .NET Core.
 
+* Windows Presentation Foundation (WPF) and Windows Forms: WPF and Windows Forms applications are only available on the .NET Framework. There are no plans to port them to .NET Core, and they remain part of the .NET Framework. 
```

Comment:
I'd still recommend striking ", and they remain part of the .NET Framework", since it's stated in the first sentence. cc/ @mairaw